docs/how-to/join-a-network.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Joining a network
+sidebarDepth: 0
+description: Learn how to join one of the various Filecoin testnets.
+---
+
+# Joining a network
+
+Once a Lotus node has been initialised on the appropriate [Filecoin network option](https://docs.filecoin.io/how-to/networks/), the instructions within this section will 
+
+### Note: Using the Lotus Node from China
+
+If you are trying to use `lotus` from China, set this **environment variable** on your machine:
+
+```sh
+IPFS_GATEWAY="https://proof-parameters.s3.cn-south-1.jdcloud-oss.com/ipfs/"
+```
+
+## Get started
+
+Start the **daemon** using the default configuration in `./build`:
+
+```sh
+lotus daemon
+```
+
+In another terminal window, check your connection with peers:
+
+```sh
+lotus net peers | wc -l
+```
+
+In order to connect to the network, you need to be connected to at least 1 peer. If you’re seeing 0 peers, read our [troubleshooting instructions](https://docs.filecoin.io/mine/mining-troubleshooting/).
```

Comment:
I believe this currently isn't in this repo's troubleshooting steps. Can you migrate that content over too?

Please confirm for all other mentions of "Troubleshooting" in this page as well.